go/vt/vttablet/onlineddl/executor.go

```diff
@@ -1071,6 +1087,16 @@ func (e *Executor) cutOverVReplMigration(ctx context.Context, s *VReplStream, sh
 	}
 	go log.Infof("cutOverVReplMigration %v: stopped vreplication", s.workflow)
 
+	defer func() {
+		if !renameWasSuccessful {
+			// Restarting vreplication
+			if err := e.startVreplication(ctx, tablet.Tablet, s.workflow); err != nil {
+				log.Errorf("cutOverVReplMigration %v: failed restarting vreplication after cutover failure: %v", s.workflow, err)
+			}
+			go log.Infof("cutOverVReplMigration %v: started vreplication after cutover failure", s.workflow)
```

Comment:
why to start a goroutine for calling in log function?

Reply:
@dbussink , yeah, I'd like this entry to appear in the logs. If it appears, it means vreplication has definitely started, which is an important info